package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Micro-service used by NEARStudio to deploy smart contracts.",
   "main": "app.js",
   "scripts": {
-    "start": "env $(sed 's/ # .*//' .env) supervisor app | bunyan",
+    "start": "node scripts/run-development-dynamo.js & env $(sed 's/ # .*//' .env) supervisor app | bunyan",
```

Comment:
Since we don't use `yarn start` in production environments this should be fine here.